src/main/build/debug.c

```diff
@@ -85,9 +85,9 @@ const char * const debugModeNames[DEBUG_COUNT] = {
     "AC_ERROR",
     "DUAL_GYRO_SCALED",
     "DSHOT_RPM_ERRORS",
-    "CRSF_LS_UPLINK",
-    "CRSF_LS_PWR",
-    "CRSF_LS_DOWN",
+    "CRSF_LINK_STATISTICS_UPLINK",
+    "CRSF_LINK_STATISTICS_PWR",
+    "CRSF_LINK_STATISTICS_DOWN",
```

Comment:
Might pay to look into adding a test to assert the maximum length in a separate pull request.

Reply:
That makes sense. Although the code can handle up to 30 characters so the risk is low.